client/js/app/main/common/taskList/taskItem.directive.controller.js

```diff
@@ -0,0 +1,17 @@
+/*
+* @ngInject
+* */
+function TaskItemController() {
+    this.isSelected = false;
```

Comment:
why we need two properties for same things? 

Looks like _selected and isSelected are same